base/essentials.jl

```diff
@@ -147,7 +147,7 @@ macro specialize(vars...)
 end
 
 """
-    @isdefined s::Bool
+    (@isdefined s)::Bool
```

Comment:
Sorry to hold things up, but maybe this'd be better written as `@isdefined(s)::Bool`? Not sure it's a huge deal either way, but might look a little less weird.

Reply:
```suggestion
    @isdefined(s)::Bool
```
Sounds good to me, thanks for the review!